galley/pkg/config/analysis/analyzers/injection/injection.go

```diff
@@ -73,8 +86,11 @@ func (a *Analyzer) Analyze(c analysis.Context) {
 		_, okNewInjectionLabel := r.Metadata.Labels[RevisionInjectionLabelName]
 
 		if injectionLabel == "" && !okNewInjectionLabel {
-			// TODO: if Istio is installed with sidecarInjectorWebhook.enableNamespacesByDefault=true
+			// if Istio is installed with sidecarInjectorWebhook.enableNamespacesByDefault=true
 			// (in the istio-sidecar-injector configmap), we need to reverse this logic and treat this as an injected namespace
+			if enableNamespacesByDefault {
+				return true
```

Comment:
we might need to tell user injecition is enabled by this flag on all namespaces.

Reply:
done, added a new message type named `NamespaceInjectionEnabledByDefault` to report this information to user.